extended_templates/compat.py

```diff
@@ -203,3 +203,11 @@ def is_authenticated(request):
     if callable(request.user.is_authenticated):
         return request.user.is_authenticated()
     return request.user.is_authenticated
+
+
+try:
+    from django.core.files.storage import storages # Added in Django 4.2
+    def get_storage_class():
+        return import_string(storages.backends['default']['BACKEND'])
+except ImportError:
+    from django.core.files.storage import get_storage_class # Removed in Django 5.0
```

Comment:
Curious, what is the rationale from the Django team to remove `get_storage_class`?

Reply:
I'm not sure. There didn't seem to be an obvious reason to get rid of it.